target/linux/ar71xx/image/legacy-devices.mk

```diff
@@ -434,6 +434,11 @@ define LegacyDevice/WPE72_16M
 endef
 LEGACY_DEVICES += WPE72_16M
 
+define LegacyDevice/WNR2000
```

Comment:
Yeah, blogic is right. Missed that using the new build code means that the NetgearLzma recipe need to be converted to the new image build code. And that is indeed out of scope of this patch.